README.md

```diff
@@ -1,11 +1,11 @@
 # Custom Template for LifterLMS #
-**Contributors:** [brainstormforce](https://profiles.wordpress.org/brainstormforce), [wpcrafter](https://profiles.wordpress.org/wpcrafter)  
+**Contributors:** brainstormforce, wpcrafter  
```

Comment:
@nileshc-bsf I think links should not be removed.

Reply:
Correct, let me check

Reply:
@premanshup - Updated the package.json file and npm updated.

Now the links are present.